cgit/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM clearlinux/httpd
 
-ARG swupd_args
+ARG swupd_args="--no-boot-update $swupd_args"
```

Comment:
Hmm, I wouldn't expect this to work, since `swupd_args` references itself? It would helpful to test whether additional swupd args can be passed using this syntax.

Reply:
I tried, this way is not worked as expected. Besides, we actually just want/need to pass "--no-boot-update" to "swupd update" not "swupd bundle-add", so changing swupd_args may not be the best choice. Maybe just add "--no-boot-update" as the parameter for the "swupd update" would be fine.

Reply:
Thanks for testing.

@gtkramer Can you update to hardcode `--no-boot-update` for update commands?

Reply:
@phmccarty , @qzheng527 I think we also want to add this to bundle-add operations too because after chatting with the swupd lead, they will also trigger a boot update with CBM.  I was looking for a way to do this without defining an ENV variable such as swupd_default_args='--no-boot-update' and then put that in every swupd command.

Reply:
@gtkramer perhaps something like this will work?

```
ARG swupd_args
ENV swupd_args --no-boot-update ${swupd_args}
```